### DISCUSSION TITLE
Rustdesk Client Installer Setting

### DISCUSSION THREAD

**Discussion by Nollknolle** [rustdesk/rustdesk#2301] [Q&A]
Hello,
Is there a way to configure the ID/connection server in the installer or do I always have to do it manually in the client?

Thanks in advance!

**Comment by pedjas** [comment]
Do not use installer. Use standalone exe (remove installed version). 
Then you can set connection parameters in exe name.

**Reply by Nollknolle** [reply]
Can you show me which options I can configure?

Thanks in advance!

**Reply by pedjas** [reply]
> Can you show me which options I can configure?

Check this: https://github.com/rustdesk/rustdesk/discussions/2152
